7Cav-Alive.Altis/scripts/missionEnd/missionEnd_air.sqf

```diff
@@ -0,0 +1,55 @@
+//Set a variable in the missionNameSpace to EndMissionNumber and set boxNumber equal to it.
+//We do this so that, if a previous 
+private _boxNumber = missionNameSpace getVariable "endMissionNumber";
+private _term = Term1;
+removeAllActions _term;
+[_term,1] call BIS_fnc_DataTerminalAnimate; //keyframe 1 
+
+
+//Create task
+private _title = "Defend The Satellite!";
+private _description = "Activate the sattellite array to destroy the armor commander!";
+private _waypoint = "_term";

```

Comment:
You define a variable for this, but then only use it on the next line. You might as well just use the actual value on the next line and make it easier to read.

Reply:
its used in several places, so instead of changing them all I just change it on the one line.

Reply:
Its not used anywhere else in this file

Reply:
> private _endMission = [BLUFOR, "Defend The Satellite!", ["Activate the satellite array to destroy the armor commander!", "Defend The Satellite!", "_term"], Term1, true] call BIS_fnc_taskCreate; //create the task (google it, it's under taskTutorial)

reduced to this